data_handler/db/models.py

```diff
@@ -98,3 +98,17 @@ class HealthRatioLevel(Base):
     user_id = Column(String, index=True)
     value = Column(DECIMAL, nullable=False)
     protocol_id = Column(ChoiceType(ProtocolIDs, impl=String()), nullable=False)
+
+
+class ZkLendCollateralDebt(Base):
+    """
+    SQLAlchemy model for the liquidable debt table for ZkLend.
```

Comment:
We dont store  liquidable debt in this table, right? Seems more like obligation data.

Reply:
yeah, we store it in different table. Liquidable debt works as standalone entity

Reply:
```suggestion
    SQLAlchemy model for table with obligation data for ZkLend.
```